thunx-spring/src/main/java/org/springframework/data/jpa/repository/support/AbacQuerydslJpaPredicateExecutor.java

```diff
@@ -0,0 +1,44 @@
+package org.springframework.data.jpa.repository.support;
+
+import com.querydsl.core.types.EntityPath;
+import com.querydsl.core.types.Predicate;
+import com.querydsl.core.types.dsl.PathBuilder;
+import com.querydsl.jpa.JPQLQuery;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.querydsl.EntityPathResolver;
+import org.springframework.data.repository.support.PageableExecutionUtils;
+import org.springframework.util.Assert;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Query;
+
+public class AbacQuerydslJpaPredicateExecutor<T> extends QuerydslJpaPredicateExecutor<T> {
```

Comment:
Is this custom querydsl-predicate-executor still directly tied to abac/thunx ?
Or can we _promote_ this to an alternative querydsl-predicate-executor, that can be used wether we're using thunx or not ?

Reply:
No, it isn't really tied to thunx at all.  Its definitely an orthogonal concern.  If you wanted to move it out and into some other xenit repo that would work I think.